src/chainstate/stacks/boot/costs-2.clar

```diff
@@ -1 +1,567 @@
+
+(define-read-only (cost_index_of (n uint))
+    (runtime (linear n u1 u243)))
+
+(define-read-only (cost_fold (n uint))
+    (runtime u483))
+
+(define-read-only (cost_list_cons (n uint))
+    (runtime (linear n u14 u198)))
+
+(define-read-only (cost_type_parse_step (n uint))
+    (runtime u5))
+
+(define-read-only (cost_tuple_get (n uint))
+    (runtime (nlogn n u4 u1780)))
+
+(define-read-only (cost_tuple_merge (n uint))
+    (runtime (linear n u4 u646)))
+
+(define-read-only (cost_tuple_cons (n uint))
+    (runtime (nlogn n u11 u1101)))
+
+(define-read-only (cost_add (n uint))
+    (runtime (linear n u12 u156)))
+
+(define-read-only (cost_sub (n uint))
+    (runtime (linear n u12 u156)))
+
+(define-read-only (cost_mul (n uint))
+    (runtime (linear n u14 u157)))
+
+(define-read-only (cost_div (n uint))
+    (runtime (linear n u14 u157)))
+
+(define-read-only (cost_geq (n uint))
+    (runtime u166))
+
+(define-read-only (cost_leq (n uint))
+    (runtime u166))
+
+(define-read-only (cost_le (n uint))
+    (runtime u166))
+
+(define-read-only (cost_ge (n uint))
+    (runtime u166))
+
+(define-read-only (cost_int_cast (n uint))
+    (runtime u164))
+
+(define-read-only (cost_mod (n uint))
+    (runtime u168))
+
+(define-read-only (cost_pow (n uint))
+    (runtime u170))
+
+(define-read-only (cost_sqrti (n uint))
+    (runtime u167))
+
+(define-read-only (cost_log2 (n uint))
+    (runtime u161))
+
+(define-read-only (cost_xor (n uint))
+    (runtime u167))
+
+(define-read-only (cost_not (n uint))
+    (runtime u162))
+
+(define-read-only (cost_eq (n uint))
+    (runtime (linear n u7 u172)))
+
+(define-read-only (cost_begin (n uint))
+    (runtime u202))
+
+(define-read-only (cost_hash160 (n uint))
+    (runtime (linear n u1 u201)))
+
+(define-read-only (cost_sha256 (n uint))
+    (runtime (linear n u1 u100)))
+
+(define-read-only (cost_sha512 (n uint))
+    (runtime (linear n u1 u176)))
+
+(define-read-only (cost_sha512t256 (n uint))
+    (runtime (linear n u1 u188)))
+
+(define-read-only (cost_keccak256 (n uint))
+    (runtime (linear n u1 u221)))
+
+(define-read-only (cost_secp256k1recover (n uint))
+    (runtime u14344))
+
+(define-read-only (cost_secp256k1verify (n uint))
+    (runtime u13540))
+
+(define-read-only (cost_print (n uint))
+    (runtime (linear n u3 u1413)))
+
+(define-read-only (cost_some_cons (n uint))
+    (runtime u219))
+
+(define-read-only (cost_ok_cons (n uint))
+    (runtime u219))
+
+(define-read-only (cost_err_cons (n uint))
+    (runtime u219))
+
+(define-read-only (cost_default_to (n uint))
+    (runtime u249))
+
+(define-read-only (cost_unwrap_ret (n uint))
+    (runtime u299))
+
+(define-read-only (cost_unwrap_err_or_ret (n uint))
+    (runtime u339))
+
+(define-read-only (cost_is_okay (n uint))
+    (runtime u287))
+
+(define-read-only (cost_is_none (n uint))
+    (runtime u212))
+
+(define-read-only (cost_is_err (n uint))
+    (runtime u282))
+
+(define-read-only (cost_is_some (n uint))
+    (runtime u223))
```

Comment:
I think `is_ok` should have same cost as `is_err`, and `is_none` same as `is_none`.

Reply:
Yep -- fixed.